ztag/schema.py

```diff
@@ -532,6 +532,31 @@ def ztag_zgrab2_transformed(service, results):
                              "the handshake.")
 }))
 
+ztag_mongodb = ztag_zgrab2_transformed(service="MongoDB", results=SubRecord({
+        "build_info": SubRecord({
+            "version": String(doc="Version of mongodb server"),
+            "git_version": String(doc="Git Version of mongodb server"),
+            "max_wire_version": Signed32BitInteger(),
+            "build_environment": SubRecord({
+                "dist_mod": String(),
+                "dist_arch": String(),
+                "cc": String(),
+                "cc_flags": String(),
+                "cxx": String(),
+                "cxx_flags": String(),
+                "link_flags": String(),
+                "target_arch": String(),
+                "target_os": String()})}),
+        "is_master": SubRecord({
```

Comment:
Assuming this is only present on masters, I'd just call this `master`

Reply:
The is_master block is the result of issuing the "isMaster" command, which returns a boolean "is_master" field which is true if the server is a "master", in addition to the rest of this info. Confirmed in the 1 percent scans that this can be false and the rest of the fields are still present.

```
{"is_master":{"is_master":false,"max_wire_version":6,"max_bson_object_size":16777216,"max_write_batch_size":100000,"logical_session_timeout_minutes":30,"max_message_size_bytes":48000000,"read_only":false}}```

Reply:
https://docs.mongodb.com/manual/reference/command/isMaster/

Reply:
note IRL - add doc field on is_master block